packages/js/product-editor/src/blocks/schedule-sale/edit.tsx

```diff
@@ -99,12 +106,17 @@ export function Edit( {}: BlockEditProps< ScheduleSalePricingBlockAttributes > )
 					);
 				}
 			}
-		}
+		},
+		[ showScheduleSale, dateOnSaleFromGmt, _dateOnSaleFrom, _dateOnSaleTo ]
 	);
 
-	const dateOnSaleToGmtValidationError = useValidation(
-		'product/date_on_sale_to_gmt',
-		function dateOnSaleToValidator() {
+	const {
+		// ref: dateOnSaleToGmtRef,
```

Comment:
Same question on this one.

Reply:
That is because the DateTimePickerControl is not ForwardedRef yet. That is going to be covered in [#38248](https://github.com/woocommerce/woocommerce/issues/38248).